app/screens/channel_add_members/channel_add_members.tsx

```diff
@@ -133,6 +135,10 @@ export default function ChannelAddMembers({
     const [term, setTerm] = useState('');
     const [addingMembers, setAddingMembers] = useState(false);
     const [selectedIds, setSelectedIds] = useState<{[id: string]: UserProfile}>({});
+    const [showBanner, setShowBanner] = useState(true);
+
+    // Use the hook to fetch access control attributes
+    const {attributeTags} = useAccessControlAttributes('channel', channel?.id, channel?.abacPolicyEnforced);
```

Comment:
these attributes should not be ephemeral

Reply:
for now, they are.